industrial_ci/src/tests/source_tests.sh

```diff
@@ -25,11 +25,18 @@
 ici_require_run_in_docker # this script must be run in docker
 
 #Define some verbose env vars
+#verbose build
 if [ "$VERBOSE_OUTPUT" ] && [ "$VERBOSE_OUTPUT" == true ]; then
     OPT_VI="-vi"
 else
     OPT_VI=""
 fi
+#verbose run tests
+if [ "$VERBOSE_TESTS" ] && [ "$VERBOSE_TESTS" == true ]; then
+    OPT_RUN_V="-v"
+else
+    OPT_RUN_V=""
+fi
 
```

Comment:
```
if [ "$VERBOSE_TESTS" = false ]; then
    OPT_RUN_V=""
else
    OPT_RUN_V="-v"
fi
```

Reply:
😉 I thought it'd be more complicated ^^
(but I'll wait until the decision with/without `interleave` is made)